ITEM.tsv

```diff
@@ -2191,7 +2191,7 @@ ITEM_20150317_002190	A tooth hard enough to bite through strong wood. Obtained f
 ITEM_20150317_002191	Sauga Core 
 ITEM_20150317_002192	A mutant variation of the Nucl, the power source of mutant monsters. Contains highly condensed magic that might explode if handled carelessly. Obtained from Saugas. 
 ITEM_20150317_002193	Lauzinute Skin 
-ITEM_20150317_002194	It is stronger than Zinute Skin. Obtained from Lauzinutes. 
```

Comment:
"Stronger than Zinute Skin" is better since it probably is making a comparison.
Like saying "Snake skin is tougher than Armadillo skin" or something